airbyte-webapp/src/components/connection/ConnectionOnboarding/ConnectionOnboarding.tsx

```diff
@@ -241,7 +241,7 @@ export const ConnectionOnboarding: React.FC<ConnectionOnboardingProps> = ({ onCr
         </div>
       </div>
       <div className={styles.footer}>
-        <Button onClick={() => onCreate()} size="lg" data-id="new-connection">
+        <Button onClick={() => onCreate()} size="lg" data-id="new-connection" data-testid="new-connection-button">
```

Comment:
is `data-id` still important? What is it being used for currently?

Reply:
I've double-checked, seems like we don't use it at all, so I removed it.